HHS/Head-Start-TTADP
# [TTAHUB-713] Fix some modal accessibility issues

## Description of change

Radio buttons were not being read incorrectly in our modal (goal change/suspend modal specifically).
Investigation revealed that the entire document was having aria-hidden appended at the root when the modal was open.
Current versions of our component library render the the modal in a portal, so I updated our library. Some additional fixes were needed to get everything to agree, and I did some general refactoring to generate less complex HTML.

## How to test

Since this updated a base dependency, please review core activities to make sure everything still functions properly.
Review the modals, since they had the most code changed. Lastly, confirm that the radio buttons are read off correctly within the goal change reason modal.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-713

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
